compiler/natives/src/sync/sync.go

```diff
@@ -14,11 +14,20 @@ func runtime_Semacquire(s *uint32) {
 }
 
 // SemacquireMutex is like Semacquire, but for profiling contended Mutexes.
-// Mutex profiling is not supported, so just use the same implementation.
+// Mutex profiling is not supported, so just use the same implementation as runtime_Semacquire.
 // TODO: Investigate this. If it's possible to implement, consider doing so, otherwise remove this comment.
-var runtime_SemacquireMutex = runtime_Semacquire
+func runtime_SemacquireMutex(s *uint32, lifo bool) {
+	// TODO: Use lifo if needed/possible.
+	if *s == 0 {
```

Comment:
I was a little worried about increasing the call stack by 1 more level in something so low-level, which is why I copied the implementation with a comment. But `runtime_SemacquireMutex` should be rarely used, since mutex profiling isn't supported. So it should be fine. Done.